lbrynet/daemon/Daemon.py

```diff
@@ -1053,6 +1053,8 @@ def jsonrpc_status(self, session_status=False):
                     'blocks': local blockchain height,
                     'blocks_behind': remote_height - local_height,
                     'best_blockhash': block hash of most recent block,
+                    'headers_download_progress': the download progress of the blockchain_headers file,
```

Comment:
Maybe show only that? While testing I noticed it goes back to 0 when download is skipped (unnecessary).
I would suggest something like (programming in comments reads bad, feel free to implement it readable):
`headers_download_progress = headers_download_progress if is_downloading_headers else ((local_height/remote_height) if remote_height else 100)`

Reply:
I thought about this initially, and having the `is_downloading_headers` flag would help with differentiating between a first time run (no `blockchain_headers` file present), and a subsequent run (when it's trying to catch up with blocks).